pimd/pim_cmd_common.h

```diff
@@ -20,6 +20,22 @@
 #ifndef PIM_CMD_COMMON_H
 #define PIM_CMD_COMMON_H
 
+void json_object_pim_upstream_add(json_object *json, struct pim_upstream *up);
```

Comment:
I suppose that in this commit there is no change and just moving the functions from one file to the other as it is.

Reply:
yes mobash